install_local.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Helper script for installing spyder and external-deps locally in develop mode.
+"""
+import argparse
+import os
+import re
+import sys
+from subprocess import check_output
+from pathlib import Path
+
+DEVPATH = Path(__file__).resolve().parent
+DEPS_PATH = DEVPATH / 'external-deps'
+INSTALL_COMMAND = [sys.executable, '-m', 'pip', 'install',
+                   '--no-deps', '-e', '.']
+
+
+def install_spyder():
+    # Install spyder locally
+    print("*. Installing spyder locally")
+    check_output(INSTALL_COMMAND, cwd=DEVPATH)
+
+
+def install_spyder_kernels():
+    # Install spyder-kernels locally
+    print("*. Installing spyder-kernels locally")
+    check_output(INSTALL_COMMAND, cwd=DEPS_PATH / 'spyder-kernels')
+
+
+def install_qdarkstyle():
+    # Install qdarkstyle locally
+    print("*. Installing qdarkstyle locally")
+    check_output(INSTALL_COMMAND, cwd=DEPS_PATH / 'qdarkstyle')
+
+
+def get_python_lsp_version():
+    """Get current version to pass it to setuptools-scm."""
+    req_file = DEVPATH / 'requirements' / 'conda.txt'
+    with open(req_file, 'r') as f:
+        lines = f.readlines()
+
+    for line in lines:
+        if 'python-lsp-server' in line:
+            # Get version part of dependency line
+            version = line.strip().split()[1]
+
+            # Get lower bound
+            version = version.split(',')[0]
+
+            # Remove comparison signs and only leave version number
+            version = re.search(r'\d+.*', version).group()
+
+    return version
+
+
+def install_python_lsp_server():
+    # Install PyLSP locally
+    print("*. Installing PyLSP locally")
+    check_output(
+        INSTALL_COMMAND, cwd=DEPS_PATH / 'python-lsp-server',
+        env={**os.environ,
+             **{'SETUPTOOLS_SCM_PRETEND_VERSION': get_python_lsp_version()}
+             }
+    )
+
+
+_installers = {'spyder': install_spyder,
+               'spyder-kernels': install_spyder_kernels,
+               'qdarkstyle': install_qdarkstyle,
+               'python-lsp-server': install_python_lsp_server}
+
+if __name__ == '__main__':
+    # ---- Parse command line
+
+    parser = argparse.ArgumentParser(usage="python install_local.py [options]")
+    parser.add_argument(
+        '--install', dest='install_list',
+        default='spyder,spyder-kernels,qdarkstyle,python-lsp-server',
```

Comment:
```suggestion
        default=','.join(_installers.keys()),
```

Avoid duplication here